pages/news/2020-10.md

```diff
@@ -0,0 +1,142 @@
+is difficult without a reliance on automation. Automation, in turn, requires
+strong consistency.
+
+While ordinarily AIPs stay in review for about a month, we intend for this
+review cycle to be longer, to allow time for APIs to be released that follow
+this guidance. Assuming the feedback is sufficiently positive, we will probably
+be looking at formal approval around the end of the year.
+
+## AIPs under review
+
+### AIP-128: Declarative-friendly interfaces
+
+> Many services need to interact with common DevOps tools, particularly those
+> that create and manage network-addressible resources (such as virtual
+> machines, load balancers, database instances, and so on). These tools revolve
+> around the principle of "configuration as code": the user specifies the
+> complete intended landscape, and tooling is responsible for making whatever
+> changes are necessary to achieve the user's specification.
+>
+> These tools are **declarative**: rather than specifying specific actions to
+> take, they specify the desired _outcome_, with the actions being derived
+> based on the differences between the current landscape and the intended one.
+
+[Declarative-friendly interfaces][aip-128] describes patterns for making
+interfaces that are friendly to automated tooling. The basic idea is that by
+being more strict and consistent, it becomes possible to write tools that
+"translate" the interface to a declarative form without having to know anything
+about the specific API.
+
+This reduces feature latency for users. Essentially, the adapter or plugin can
+be automatically generated, and so an automated pipeline becomes possible: as
+soon as the service launches new features, they become available in the
+adapters also.
+
+Because declarative-friendliness is a wide-ranging concept, impacting the
+guidance in numerous other AIPs as well, it was difficult to determine how to
+structure the information. Therefore, this AIP also has a robust "further
+reading" section that points to declarative-friendly guidance in other AIPs.
+
+**Summary:** [AIP-128][] provides an explanation of what it means for a service
+to be declarative-friendly, and why a service might want to adopt these
+patterns. It also indexes declarative-friendly guidance found in other AIPs.
+
+[aip-128]: /128
+
+### AIP-148: Standard fields
+
+> Certain concepts are common throughout any corpus of APIs. In these
+> situations, it is useful to have a standard field name that is used
+> consistently to communicate that concept.
+
+[Standard fields][aip-148] is, in some ways, an overdue AIP. It describes
+fields common throughout our corpus, such as `name`, `parent`, and
+`create_time`, that were on the list of standard fields in the original API
+style guide and did not get a full treatment in other AIPs. Common fields that
+are covered in other AIPs are not discussed in this one. (As an example,
+`language_code` is given extensive treatment in [AIP-143][], so not repeated
+here.)
```

Comment:
Do we *list* all the standard fields in AIP-148, with links for further reading? If we don't, I think we should - it's good to have a canonical list somewhere that's easy to scan through.

Reply:
The further reading section in AIP-148 was unintentionally empty. Fixed.